SegmentedControlTab.js

```diff
@@ -34,14 +34,17 @@ const TabOption = ({
             firstTabStyle,
             lastTabStyle]}
             onPress={() => onTabPress(index)}
-            activeOpacity={1}>
-            <View style={{ flexDirection: "row" }}>
+            activeOpacity={1}
+            accessibilityLabel={accessibilityLabel + "_touchableOpacity"}>
```

Comment:
should be: accessibilityLabel={accessibilityLabel}  

@kirankalyan5  Thoughts?

Reply:
True, so we give the liberty for the consumer to set the `accessibilityLabel`. 

This issue will be soon fixed as a part of #55 PR.